### DISCUSSION TITLE
Open source annotation software

### DISCUSSION THREAD

**Discussion by Bwwwah** [ultralytics/yolov5#7073] [Q&A]
Hello everyone ! I wanted to know if anyone knows an open source annotation software compatible with this version of yolo? It would help me a lot because I keep looking and I can't find. Thank you !

**Comment by glenn-jocher** [comment]
@Bwwwah you could try https://github.com/SkalskiP/make-sense

**Reply by Bwwwah** [reply]
I am an apprentice in a company. Can I use this software in this specific case?

**Reply by glenn-jocher** [reply]
Sure! For the easiest annotation and dataset export to YOLOv5 though I would probably recommend [roboflow](https://roboflow.com/?ref=ultralytics). Not open-source but very easy to use.

**Reply by Bwwwah** [reply]
Ok thank you very much !

**Reply by xoxolo** [reply]
You can use also https://www.makesense.ai/
Really easy to use and effective!
O
